### DISCUSSION TITLE
[Bug] Backend log out

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#400] [Help] [answered]
After logout (... (bottom left) > Log out), login mask is shown but URL says https://xxx.swell.store/login But refreshing or directly call /login leads to a server error 500. Should be consistent.

**Comment by logeshswell** [comment]
This issue is no longer present now. Can you check and confirm?

**Comment by josias-hosang** [comment, marked as answer]
Looks fine now. Thanks.
